src/Selenium2Library/keywords/_browsermanagement.py

```diff
@@ -331,6 +331,18 @@ def select_window(self, locator=None):
             pass
         finally:
             self._window_manager.select(self._current_browser(), locator)
+            
+    def get_log(self, log_type):
+        """Get the log for a given selenium log type
```

Comment:
The Python documentation for this method is pretty inadequate. But we should be better. The information should include most of the content from the Javadoc. Also how to enable logging and setting levels would be nice to see.

> Fetches available log entries for the given log type. Note that log buffers are reset after each call, meaning that available log entries correspond to those entries not yet returned for a given log type. In practice, this means that this call will return the available log entries since the last call, or from the start of the session. For more info on enabling logging, look at LoggingPreferences.